jcommune-model/src/main/java/org/jtalks/jcommune/model/dto/GroupAdministrationDto.java

```diff
@@ -67,4 +77,10 @@ public void setNumberOfUsers(long numberOfUsers) {
         this.numberOfUsers = numberOfUsers;
     }
 
+    public boolean isEditable() {
+        //TODO
+        // Implement logic that evaluates is group editable. Something like this:
+        // Arrays.asList("Administrators", "Registered Users", "Banned Users").contains(name)
+        return editable;
```

Comment:
This is a good example of how TDD could work - you could already write a unit test for this.